1-exercises/E-for-loop/exercise2.js

```diff
@@ -27,6 +27,14 @@ const AGES = [
 ];
 
 // TODO - Write for loop code here
+function nameAndAge(WRITERS, AGES) {
+    for (let i = 0; i < WRITERS.length; i++) {
+        for (let j = 0; j < AGES.length; j++) {        
+        }
+    }
+    return (`${WRITERS[i]} is${AGES[j]} years old`);
+}
```

Comment:
1. We don't need the second for loop
2. we need to put `console.log` with the line `35` without any return
3. After `is` you need a space